### ISSUE TITLE
The word " हिन्दी " (Hindi) is duplicated in the tools GL dropdown lists, scripture pane resource selector, and project card

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/translationCore#5151]
1.0.0 (4bd6856) Release 1.0
and 1.0.0 (ea3713f) Develop

It appears that the resource may be improperly labeled.
Also, is the correct Hindi resource being used to populate the tool cards dropdown lists?

![image](https://user-images.githubusercontent.com/8171759/46442086-26e89580-c72e-11e8-9df7-9f66418fedfc.png)

![image](https://user-images.githubusercontent.com/8171759/46442267-a5ddce00-c72e-11e8-9f31-8e453e1804ab.png)

![image](https://user-images.githubusercontent.com/8171759/46442290-bc842500-c72e-11e8-8d72-e85976497da2.png)

![image](https://user-images.githubusercontent.com/8171759/46442301-cefe5e80-c72e-11e8-9895-07865712ccf2.png)

![image](https://user-images.githubusercontent.com/8171759/46443659-1fc48600-c734-11e8-912f-5b5d9e68bcde.png)

**Comment by PhotoNomad0** [comment]
@cckozie @benjore - ~the bug is that the entry in the translationDatabase (langnames.json) has "हिन्दी, हिन्दी" as the localized language name.~

**Comment by PhotoNomad0** [comment]
@cckozie @benjore - looking closer this language name is not a repeat or duplicate.   The hi entry in the translationDatabase (langnames.json) has "हिन्दी, हिंदी" as the localized language name.  Looks like there are two alternate spellings or names.  @elsylambert - does this look right to you?

**Comment by elsylambert** [comment]
@PhotoNomad0 you are right. Hindi can be written in both of those spellings. however, हिन्दी is the one most commonly written. We can stick to one spelling.

**Comment by cckozie** [comment]
Do we want to change this so that it's consistent everywhere?
